schemas/dns.yml

```diff
@@ -122,12 +123,39 @@
       type: keyword
       short: The subdomain of the domain.
       description: >
-        The subdomain is all of the labels under the registered_domain.
+        The subdomain portion of a fully qualified domain name includes all of the names except 
+        the host name under the registered_domain.  In a partially qualified domain, or if the 
+        the qualification level of the full name cannot be determined, subdomain contains all of
+        the names below the registered domain.
+        
+        For example the subdomain portion of "www.east.mydomain.co.uk " is "east".
 
         If the domain has multiple levels of subdomain, such as "sub2.sub1.example.com",
         the subdomain field should contain "sub2.sub1", with no trailing period.
-      example: www
+      example: east
+
+    - name: is_internationalized
```

Comment:
I like the idea of `is_internationalized`, and I think Andrew's use case from elastic/beats#5809 makes a lot of sense. Do we have a mechanism that will be able to populate this field today or in the near future?

Just wonder if we should perhaps hold it back until we have a more immediate use. Going along with the internationalized domain name use case, it'd be more useful to a user if the punycode-to-unicode translation was captured as well as setting the `is_internationalized` flag.

Reply:
found a simple regex to check   ^(.+[-_.])??xn--), contains ".xn--" in the registered / tld would also do it

From a machine learning standpoint I think the unicode translation would definitely be interesting (looks like apple, but its not!) but the is_internationalized flag seems easy enough to implement.